esmerald/responses/encoders.py

```diff
@@ -17,15 +20,48 @@ class ORJSONResponse(BaseJSONResponse):
     In the same way the JSONResponse is used, so is the `ORJSONResponse`.
     """
 
+    @staticmethod
```

Comment:
Why is this here? The transform should be as it was before. In the Response which is the base class of all of them. This is over complicating. No reason